### PR TITLE
chore: add missing data-cy ids

### PR DESCRIPTION
**Pull request by martinkaintas**
This PR adds missing data-cy ids requested by @dvdbak

### CODE REVIEW

File: src/popup/components/AccountDetailsBase.vue [hunk 1]
```diff
@@ -1,5 +1,8 @@
 <template>
-  <div class="account-details">
+  <div
+    class="account-details"
+    :data-account-address="activeAccount.address"
```
Comment (CedrikNikita) [inline src/popup/components/AccountDetailsBase.vue:4]: Why isn't it a `data-cy`?

Comment (martinkaintas) [reply src/popup/components/AccountDetailsBase.vue:4]: It could be, but given that it is not an id I think it's better to give it a more descriptive name

**Review by CedrikNikita — APPROVED** [review]

File: src/popup/components/Modals/MultisigVaultCreate.vue [hunk 2]
```diff
@@ -20,6 +20,7 @@
       <div
         v-for="(signer, index) in signers"
         :key="`signer${index}`"
+        :data-cy="`signer-address-${index}`"
```
Comment (peronczyk) [inline src/popup/components/Modals/MultisigVaultCreate.vue:23]: I strongly recommend to not use dynamic test IDs.

**Review by peronczyk — CHANGES_REQUESTED** [review]

**Review by dvdbak — APPROVED** [review]

File: src/popup/components/Modals/MultisigVaultCreate.vue (hunk 2 above)
Comment (martinkaintas) [reply src/popup/components/Modals/MultisigVaultCreate.vue:23]: I changed it to `:data-signer-idx` to make it more clear

**Review by peronczyk — APPROVED** [review]

**Pull request merged**